functions.php

```diff
@@ -65,6 +68,38 @@ function dt_pcsu_signup_blog_notification_email_rename( $message, $domain, $path
         What is your target location or people group? <span class="private-notice">Answer is kept private.</span>
     </label>
     <input type="text" id="dt_reason_for_subsite" name="dt_reason_for_subsite">
+    <p>
+        <label>Chose a Campaign Type:</label>
+            <?php
+            // @todo enable this
+            // $wizard_types = apply_filters( 'dt_campaigns_wizard_types', [] );
```

Comment:
Yeah I was just wondering about this too.
It would be nice if this was autoupdating from the filter

Reply:
At some point :D

Reply:
@squigglybob I added this as a new issue to the backlog.